### PR TITLE
Feat/job section

### PR DESCRIPTION
**Pull request by DeadLoss1801**
Adding JWT Auth & functions.
This is only for preview.

### CODE REVIEW

**Comment by DeadLoss1801** [comment]
Adding JWT Auth & functions.
This is only for preview.

File: src/index.ts [hunk 1]
```diff
@@ -29,15 +29,17 @@ database.once('connected', () => {
...
 app.use(express.json());
+// use cookie-parser
+app.use(cookieParser());
 
 // Start the server
 app.listen(3000, () => {
 	console.log('Server Started at http://localhost:3000');
 });
 
+import router from './router';
```
Comment (aka1602) [inline src/index.ts:42]: why we are importing routes here in this line

File: src/index.ts [hunk 2]
```diff
@@ -52,7 +54,7 @@ const options: Options = {
 		},
 		servers: [
 			{
-				url: 'http://localhost:8000',
+				url: 'http://localhost:3000',
```
Comment (aka1602) [inline src/index.ts:57]: try to make only one port and use it

File: src/utils/common.ts [hunk 3]
```diff
@@ -1,10 +1,53 @@
...
+	req: express.Request,
+	res: express.Response,
+	next: express.NextFunction
+) => {
+	try {
+		const token = req.cookies.login;
+		if (!token) {
+			return res
+				.json({
+					message: 'Please logged in',
```
Comment (aka1602) [inline src/utils/common.ts:26]: message grammer is incorrect

**Pull request closed without merging**